_i18n/en.yml

```diff
@@ -0,0 +1,81 @@
+---
+global:
+  brazilian portuguese: Português
+  english: English
+header:
+  stories: Stories
+  explore: Explore
+  about: About
+support:
+  fund_us: Support the project
+home:
+  about: Artificial Intelligence for social control of public administration
+  about_rosie: Operação Serenata de Amor created a Robot - An artificial intelligence capable of analysing each request for reimbursement from congresspeople and identify it’s illegal probability.
+  last_histories: Last published histories
+  loading: Loading...
+  more_posts: More posts
+footer:
+  about: about
+  github: Serenata on github
+  contact: contact
+  partners: partners
+stories:
+  published: Published stories
+  loading: Loading...
+explore:
+  reports: Reports
+  about_reports: Rosie, our Robot, in numbers
+  tweet_about: Tweet about this number
+  chamber_reports: Reports done to the Chamber of Deputies
+  suspitions: Different congresspeople reported for suspicious reimbursements
+  questionated: thousand Brazilian Reals in suspicious expenses questioned
+  paragraph_one: Those values correspond to 371% of what we raised with our crowdfunding campaign (~U$ 25.000,00)
+  paragraph_two: Want to know how the report system works?
+  more: You can read more in this post
+about:
+  title: Serenata de amor operation.
+  who_contributed: People who contributed
+  paragraph_one: We are a project who make use of artificial intelligence to audit public accounts and fight corruption. The idea came from the data scientist
+  paragraph_two:  when he realized about the many possibilities to use technology to inspect congressperson’s spends. When he shared the idea with friends, the project growth in strength in no time and the team was formed with the goal to finish the project.
+  paragraph_three: Serenata de amor Operation was born, focused on inspection of reimbursements made by Quota to the Exercise of Parliamentary Activity, who pays for alimentation, transport, lodging and even culture, courses and TV signatures of federal deputies.
+  paragraph_four: On September 7 goes live the first crowdfunding campaign of the project, asking for money to pay for the team work for 2 months. In the end, the campaign raised 20% more than the original asked value, summing up more than U$25.000,00.
```

Comment:
Some minor edits:
As we used _analyse_ somewhere I'm assuming we're embracing British English in favor of American English. This is weird because we use Dollars and not Pounds or Euros. 

The thing is that if we keep with the Britsh, dates should be _day of month_ and not _month day_. But maybe we can revert that _analyse_ to _analyze_ and keep with the American…

Besides that:

_On the 7 of September the first crowdfunding campaign for the project goes live asking for money to pay for the team work full time during 2 months. In the end the campaign raised 20% more than the original target, summing up more than 25,000.00 USD._